_inc/client/components/upgrade-notice-content/index.jsx

```diff
@@ -38,62 +42,42 @@ const UpgradeNoticeContent = withModuleSettingsFormHelpers(
 		};
 
 		renderInnerContent() {
-			const blockEditorUrl = `${ this.props.adminUrl }post-new.php`;
+			/*eslint-disable react/no-danger*/
 			return (
 				<div className="jp-upgrade-notice__content">
-					<p>
-						{ __( 'The features you rely on, adapted for the new WordPress editor.' ) }
-						<br />
-						{ __( 'A new editor? Yes! {{a}}Learn more{{/a}}.', {
-							components: {
-								a: (
-									<ExternalLink
-										target="_blank"
-										rel="noopener noreferrer"
-										href={ 'https://wp.me/p1moTy-cee' }
-									/>
-								),
-							},
-						} ) }
-					</p>
-
-					<h2>{ __( 'Build your Jetpack site with blocks' ) }</h2>
-
-					<p>
-						{ __(
-							'Today, we are introducing the first wave of Jetpack-specific blocks built specifically ' +
-								'for the new editor experience: Simple Payment button, Form, Map, and Markdown.'
-						) }
-					</p>
-					<p>
-						<img
-							src={ imagePath + 'block-picker.png' }
-							width="250"
-							alt={ __( 'Jetpack is ready for the new WordPress editor' ) }
-						/>
-					</p>
+					<div dangerouslySetInnerHTML={ { __html: this.props.releasePostContent } } />
 					<div className="jp-dialogue__cta-container">
-						<Button primary={ true } href={ blockEditorUrl } onClick={ this.trackLearnMoreClick }>
-							{ __( 'Take me to the new editor' ) }
-						</Button>
 						<Button onClick={ this.dismissNotice }>{ __( 'Okay, got it!' ) }</Button>
 					</div>
+					<br />
+					<div>
+						<ExternalLink
+							href={ this.props.releasePostLink }
+							target="_blank"
+							rel="noopener noreferrer"
+						>
+							{ __( 'View this post on the Jetpack.com blog' ) }
+						</ExternalLink>
+						<br />
+						<ExternalLink
+							href="https://wordpress.org/plugins/jetpack/#developers"
+							target="_blank"
+							rel="noopener noreferrer"
+						>
+							{ __( 'View the Jetpack ' + this.props.version + ' changelog' ) }
+						</ExternalLink>
+					</div>
 				</div>
 			);
+			/*eslint-enable react/no-danger*/
 		}
 
 		render() {
 			return (
 				// TODO: update SVG?
 				<JetpackDialogue
-					svg={
-						<img
-							src={ imagePath + 'jetpack-gutenberg.svg' }
-							width="250"
-							alt={ __( 'Jetpack is ready for the new WordPress editor' ) }
-						/>
-					}
-					title={ __( 'New in Jetpack!' ) }
+					svg={ <img src={ this.props.releasePostImage } width="350" alt={ '' } /> }
```

Comment:
Do we need the alt if we leave it empty?

Reply:
I think we do need the empty alt for accessibility. 

This is from the [Alt Text Best Practices](https://make.wordpress.org/accessibility/handbook/content/alternative-text-for-images/#best-practice) in the WordPress Accessibility Handbook:

> Give any decorative images in an img element an empty alt attribute ( alt="" ).

There's some info about why the empty alt tag is important [here](https://www.w3.org/WAI/tutorials/images/decorative/). Basically, the empty alt tag tells the screen reader that the image is decorative. If the alt tag is missing, the screen reader will announce the image's file name.